### PR TITLE
Updated Spanish (es-es.json) translations.

### PR DESCRIPTION
**Pull request by SarguCopperpot**
I improved some translations that were not very well done. Now they should be fine.

### CODE REVIEW

**Comment by AndanteDevs** [comment]
Could you also include the Liquid Tanks translations in this pull, too?

**Comment by SarguCopperpot** [comment]
Yep, I've seen that the translations of "Liquid Tanks" have been added. So I added it.

File: base/assets/gm4/lang/es_es.json [hunk 1]
```diff
@@ -64,15 +73,50 @@
...
+    "block.gm4.liquid_tank.harming": "Tanque de Poción de Daño",
+    "block.gm4.liquid_tank.healing": "Tanque de Poción de Curación",
+    "block.gm4.liquid_tank.invisibility": "Tanque de Poción de Invisibilidad",
+    "block.gm4.liquid_tank.leaping": "Tanque de Poción de Salto",
+    "block.gm4.liquid_tank.luck": "Tanque de Poción de Suerte",
+    "block.gm4.liquid_tank.night_vision": "Tanque de Poción de Visión Nocturna",
+    "block.gm4.liquid_tank.poison": "Tanque de Poción de Veneno",
+    "block.gm4.liquid_tank.regeneration": "Tanque de Poción de Regeneración",
+    "block.gm4.liquid_tank.slow_falling": "Tanque de Poción de Caída lenta",
+    "block.gm4.liquid_tank.slowness": "Tanque de Poción de Lentitud,
```
Comment (misode) [inline base/assets/gm4/lang/es_es.json]: Missing quote
```suggestion
    "block.gm4.liquid_tank.slowness": "Tanque de Poción de Lentitud",
```

File: base/assets/gm4/lang/es_es.json [hunk 2]
```diff
@@ -64,15 +73,50 @@
...
+    "block.gm4.liquid_tank.strong_healing": "Tanque de Poción de Curación (Fuerte)",
+    "block.gm4.liquid_tank.strong_leaping": "Tanque de Poción de Salto (Fuerte)",
+    "block.gm4.liquid_tank.strong_poison": "Tanque de Poción de Veneno (Fuerte)",
+    "block.gm4.liquid_tank.strong_regeneration": "Tanque de Poción de Regeneración (Fuerte)",
+    "block.gm4.liquid_tank.strong_slowness": "Tanque de Poción de Lentitud (Fuerte)",
+    "block.gm4.liquid_tank.strong_strength": "Tanque de Poción de Fuerza (Fuerte)",
+    "block.gm4.liquid_tank.strong_swiftness": "Tanque de Poción de Rapidez (Fuerte)",
+    "block.gm4.liquid_tank.strong_turtle_master": "Tanque de Poción del maestro Tortuga (Fuerte)",
+    "block.gm4.liquid_tank.swiftness": "Tanque de Poción de Rapidez",
+    "block.gm4.liquid_tank.turtle_master": "Tanque de Poción del maestro Tortuga,
```
Comment (misode) [inline base/assets/gm4/lang/es_es.json]: Missing quote
```suggestion
    "block.gm4.liquid_tank.turtle_master": "Tanque de Poción del maestro Tortuga",
```

Comment (SarguCopperpot) [reply base/assets/gm4/lang/es_es.json]: Fixed w/ the ''Mini-update 2 (Spanish Translations)'' commit.

**Pull request merged**